source/models/mock_data.json

```diff
@@ -8,6 +8,8 @@
     "collections": [{
       "type": "array",
       "name": "Collection one",
+      "logs": [
```

Comment:
What is this field for?

Reply:
`logs` contains an array of timestamps indicating which logs belong to a collection. It's used for achieving data persistence once a user adds a tag to a collection. We also considered adding a `tags` field to each daily log but doing it this way is a little more efficient.